companies/contentfleet.json

```
@@ -0,0 +1,20 @@
+{
+    "slug": "contentfleet",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Content Fleet GmbH",
+    "runs": [
+        "einfachtierisch.de"
+    ],
+    "address": "Kehrwieder 8\n20457 Hamburg",
```

```suggestion
    "address": "c/o Ströer Digital Media GmbH\nPlatz der Einheit 1\n60327 Frankfurt am Main\nDeutschland",
```
address of DPO, see [homepage](https://einfachtierisch.de/p/datenschutz)